packages/metrics/src/features/metrics/utils.ts

```diff
@@ -211,7 +216,7 @@ export const fetchEstimateByLocation = async (
       ((crudRate * femalePopulation) / 1000) * (estimationForDays / 365)
     ),
     locationId: locationData.id,
-    estimationYear: actualEstimationYear,
+    estimationYear: selectedCrudYear,
```

Comment:
Just wondering whether it should be `selectedCrudYear` or `selectedPopYear` ?

Reply:
Yes, I also wondered about this. I'm not sure if it makes any difference.  When we create the admin system for this, I think that we have to make sure that a population cannot be entered without a crude birth rate and vice versa.  In which case this code needs refactored anyway.